core/cat/factory/llm.py

```diff
@@ -96,7 +96,7 @@ class Config:
 
 class LLMOpenAIConfig(LLMSettings):
     openai_api_key: str
-    model_name: str = "text-davinci-003"
+    model_name: str = "gpt-3.5-turbo-instruct" # used instead of text-davinci-003 since it deprecated
```

Comment:
@pieroit I can't test this line, because this one refers to the model deployed to openai.com and I don't have an account to test.